gaurav/tsconfig.json

```diff
@@ -5,7 +5,7 @@
     "lib": ["ES2020", "DOM", "DOM.Iterable"],
     "module": "ESNext",
     "skipLibCheck": true,
-
+    "composite": true,
```

Comment:
Why was this added? Please explain here.

Reply:
removed `"composite": true` as it is not used anywhere in the project